File: project2/Mul.py

#Convert the following C code to Python:
# C code:
# #include <stdio.h>
# 
# int main() {
#     int a, b, mul;
#     printf("Enter two numbers: ");
#     scanf("%d %d", &a, &b);
#     mul = a * b;
#     printf("The multiplication of %d and %d is: %d\n", a,
# 
#               b, mul);
#     return 0;
# }

# Python code equivalent to the given C code
def main():
    a = int(input("Enter the first number: "))
    b = int(input("Enter the second number: "))
    mul = a * b
    print(f"The multiplication of {a} and {b} is: {mul}")

if __name__ == "__main__":
    main()

File: project6/mul.c
#include <stdio.h>

int main(void) {
    int n;
    printf("몇 단을 출력하지 입력하시오 : ");
    scanf("%d",&n);
    for (int i = 1; i < 10; i++) {
        printf("%d x %i = %d\n",n,i,n*i);
    }
    return 0;
}